projects/elife.yaml

```diff
@@ -1820,21 +1822,27 @@ digests:
             - 80:
                 cidr-ip: 10.0.0.0/16 # access via VPC ip range only
     aws-alt:
-        end2end: {}
-            #rds:
-            #    engine: postgres
-            #    storage: 5 # GB
-            #    backup-retention: 2 # days
-        continuumtest: {}
-            #rds:
-            #    engine: postgres
-            #    storage: 5 # GB
-            #    backup-retention: 2 # days
-        prod: {}
-            #rds:
-            #    engine: postgres
-            #    storage: 5 # GB
-            #    multi-az: true
+        end2end:
```

Comment:
eh, this repetition is annoying but I'm guessing you're having to do it for a good reason

Reply:
Mainly the base stack doesn't have RDS (would be used for ad-hoc stacks or for testing formulas) as it takes half the time to provision; Then it means various environments have to repeat the RDS configuration, in this case `engine`, `type`, `storage` and `storage-type` (`backup-retention` differs). A way to remove this could be defaults that are applied to certain environments e.g. a top level `default-for-environments:` key with `end2end`, `prod`, etc. subsection?

Reply:
I think the project definition logic is already too clever. I'm working on a replacement in my own time. For now, a little repetition is better than any more cleverness